lib/devise/models/confirmable.rb

```diff
@@ -48,7 +48,7 @@ module Confirmable
       included do
         before_create :generate_confirmation_token, if: :confirmation_required?
         after_create :skip_reconfirmation_in_callback!, if: :send_confirmation_notification?
-        if defined?(ActiveRecord) && self.is_a?(ActiveRecord::Base) # ActiveRecord
+        if defined?(ActiveRecord) && self < ActiveRecord::Base # ActiveRecord
```

Comment:
I think the comment `#ActiveRecord` isn't necessary. What do you think to remove it?